assets/queries/k8s/kubernetes_image_digest/query.rego

```diff
@@ -0,0 +1,38 @@
+package Cx
+
+CxPolicy [ result ] {
+	document := input.document[i]
+  metadata:= document.metadata
+  spec := document.spec
+  containers := spec.containers
+  image := containers[c].image
+  not contains(image, "@")
+    
+	
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	   sprintf("metadata.name=%s.spec.containers.name=%s.image", [metadata.name, containers[c].name]),
+                "issueType":		"IncorrectValue",
+                "keyExpectedValue": sprintf("spec[%s].containers[%s].image has '@'", [metadata.name, containers[c].name]),
+                "keyActualValue": 	sprintf("spec[%s].containers[%s].image doesn't have '@'", [metadata.name, containers[c].name])
```

Comment:
Metadata name comes before Spec, so you shouldn't search for Spec[Metadata.name]. At this point, it is not really necessary since the search key tracks down the faulty resource, but if you want to place it, do it like in the SearchKey.